simplq/src/components/pages/QueueStatus.jsx

```diff
@@ -85,6 +105,7 @@ function QueueStatus() {
       <Header text={queueName} className={styles.header} />
       <JoinerStepper />
       <div>{status}</div>
+      <NotificationButton />
```

Comment:
Both status and notificationButton are not center-aligned currently.

Whatever we use for centre aligning status gif can be applied here too.

Reply:
this is fine